docs/diagnostics/PX1038.md

```diff
@@ -5,19 +5,19 @@ This document describes the PX1038 diagnostic.
 
 | Code   | Short Description                                                                                  | Type  | Code Fix    | 
 | ------ | ---------------------------------------------------------------------------------------------------| ----- | ----------- | 
-| PX1038 | Async void methods, lambdas and anonymous methods should not be used with the Acumatica Framework. | Error | Unavailable | 
+| PX1038 | Async void methods, lambdas, and anonymous methods should not be used with the Acumatica Framework. | Error | Unavailable | 
 
 ## Diagnostic Description
-The diagnostic PX1038 reports all async void methods, lambdas and anonymous methods. 
+The diagnostic reports all async void methods, lambdas, and anonymous methods. 
 
-Usage of async void methods and lambdas is a general bad practice in the .Net programming. They can't be awaited, and exceptions thrown in them are not propagated to the caller. 
-This makes it very difficult to handle exceptions properly. The only exception is when the async void method is used as an event handler. The async void method is not awaited, and the caller does not expect a return value.
+Use of async void methods and lambdas generally is a bad practice in .Net programming. You cannot await those methods, and exceptions thrown in them are not propagated to the caller. 
+As a result, it is very difficult to handle exceptions properly. The only exception when an async void method can be used is in an event handler. The async void method is not awaited, and the caller does not expect a return value.
```

Comment:
@EAndrosova the sentence should be re-worded, because the meaning of the sentence was changed.
The async void method is used as an event handler, not in an event handler.

```suggestion
As a result, it is very difficult to handle exceptions properly. The only exception when an async void method can be used is when it is an event handler. The async void method is not awaited, and the caller does not expect a return value.
```